385/Assets/Scripts/ChangeScene.cs

```diff
@@ -1,13 +1,20 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.UI;
 using UnityEngine.SceneManagement;
 
-public class ChangeScene : MonoBehaviour {
+public class ChangeScene : MonoBehaviour 
+{
+	public static bool SelectedControllerMode = false;
```

Comment:
We should not include this to ChangeScene. We should instead have a `GlobalSettings` gameobject that does not get unloaded on scene change, or something like it to handle settings instead.

Another group sent me this video.
https://youtu.be/J6FfcJpbPXE?t=4m7s

Controller mode is not relevant to change scene. Keep the separation of concerns in mind.